### PR TITLE
Prepare for githook: isort modification of files

### PR DESCRIPTION
**Pull request by janiversen**
As part of the preparation to add a githook with the same “demands” as HA, isort was run on all python files.

This affected (nearly) all python files, but with a very change in each file.

I have not touched any of the files manually, this is the raw isort changes, which should make it easier to review.

### CODE REVIEW

**Comment by janiversen** [comment]
I knew it isort and black goes hand in hand. Let me update.

**Comment by janiversen** [comment]
In order to make review easy:
First commit is  files changed by isort (NO manual changes)
Second commit is files changed by black (due to first commit) (NO manual changes).

**Comment by janiversen** [comment]
Flake8 demands a lot of manual changes, which is more difficult to review so I will do that in a couple of PRs later.

File: pytradfri/TODO [hunk 1]
```diff
@@ -0,0 +1,35 @@
+--> Commit 2 --
```
Comment (MartinHjelmare) [inline pytradfri/TODO:1]: This file should not be included I think.

File: pytradfri/color.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
 from .const import (
     ATTR_LIGHT_COLOR_HEX,
-    ATTR_LIGHT_COLOR_X as X,
-    ATTR_LIGHT_COLOR_Y as Y,
-    ATTR_LIGHT_COLOR_SATURATION,
     ATTR_LIGHT_COLOR_HUE,
+    ATTR_LIGHT_COLOR_SATURATION,
+)
+from .const import ATTR_LIGHT_COLOR_X as X
```
Comment (MartinHjelmare) [inline pytradfri/color.py:6]: This looks weird. Please define the isort settings in a config file in the source. There's a black profile I think we can use.

File: pytradfri/TODO (hunk 1 above)
Comment (janiversen) [reply pytradfri/TODO]: ups.

File: pytradfri/color.py (hunk 2 above)
Comment (janiversen) [reply pytradfri/color.py]: Done.

**Comment by janiversen** [comment]
Major change!! using pyproject.toml with profile=black, changed the output a LOT. Now black no longer complains.

There are still pylint and flake8 things to correct, but that is in another PR.

**Review by MartinHjelmare — COMMENTED** [review]
Where are the isort settings? Please include them in the PR.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,140 @@
+[tool.black]
+target-version = ["py38"]
+exclude = 'generated'
+
+[tool.isort]
```
Comment (MartinHjelmare) [inline pyproject.toml:5]: Just add this section now.

Comment (janiversen) [reply pyproject.toml:1]: Done

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "hass_logger",
+]
+persistent = false
+extension-pkg-allow-list = [
+    "av.audio.stream",
+    "av.stream",
+    "ciso8601",
+    "cv2",
+]
+
```
Comment (MartinHjelmare) [inline pyproject.toml:39]:
```suggestion
```

Comment (janiversen) [reply pyproject.toml]: Done

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**